deployment/ansible/group_vars/packer

```diff
@@ -2,7 +2,6 @@
 django_settings_module: "mmw.settings.production"
 
 postgresql_username: "modelmywatershed"
-postgresql_password: "{{ lookup('env', 'MMW_DB_PASSWORD') | default('modelmywatershed', true) }}"
 postgresql_database: "modelmywatershed"
```

Comment:
I think just this one, because it is the only one that's sensitive enough to be outside of the group variables.